specification/wse/SPEC.md

```diff
@@ -426,12 +422,25 @@ The client requirements for data frame syntax are defined by
 The server requirements for data frame syntax are defined by 
 [Draft-76, Section 5.3](http://tools.ietf.org/html/draft-hixie-thewebsocketprotocol-76#section-5.3).
 
+Binary frames are represented using frame type 0x80.
+
+Text frames may be represented in one of two formats:
+
+* Delimited: the frame type byte is 0x00, with the 0xFF delimiter marking end of data
+* Specified length: the frame type is 0x81, which, according to the rules of the above referenced sections of [Draft-76](http://tools.ietf.org/html/draft-hixie-thewebsocketprotocol-76), implies that the length is specified at the start of frame.
```

Comment:
Correct, but 0x81 is compatible with Draft-76 because Draft-76 only talks about the leading bit of the frame type byte and poses no constraints on the remaining bits or the meaning and interpretation of the frame payload. If set then the message is specified length format.